components/issue-list-item.tsx

```diff
@@ -30,90 +29,119 @@ export default function IssueListItem({
 
-    const allReviewers = pullRequests.map((pr) => {
-      return pr.reviewers?.length;
-    });
-    allReviewers.map((num) => (numberAllReviewers = numberAllReviewers + num));
-    return numberAllReviewers;
+    if (["open", "ready", "proposal"].includes(state?.toLowerCase())) {
+      const { value, translation } = types[state?.toLowerCase()];
+      return (
+        <div className="d-flex align-items-center" key={issue.githubId}>
+          <span className="caption-medium mr-1 text-white">
+            {(issue !== null && value) || 0}
+          </span>
+          <span className="caption-medium text-white-40 text-uppercase">
+            {translation}
+          </span>
+        </div>
+      );
+    } else return null;
   }
 
-  function renderProposals() {
+  function renderAmount() {
+    const isActive = ["closed", "canceled"].includes(issue?.state);
+
+    const percentage = (issue?.amount * 100) / issue?.fundingAmount;
+
     return (
-      <div className="d-flex align-items-center" key={issue.githubId}>
-        <span className="caption-small mr-1 text-white">
-          {(issue !== null && issue?.mergeProposals?.length) || 0}
-        </span>
-        <span className="caption-small text-white-40 text-uppercase">
-          {t("info.proposals", { count: issue?.mergeProposals?.length})}
-        </span>
+      <div
+        className={`row justify-content-md-center m-0 px-1 pb-1 rounded-5 ${
+          !isActive ? "bg-black" : "bg-dark-gray"
+        } `}
+      >
+        <div className="px-0 pt-1 col-md-12">
+          <span
+            className={`caption-large text-opacity-1 text-white${
+              isActive && "-40"
+            }`}
+          >
+            {formatNumberToNScale(issue?.fundingAmount > 0 ? issue?.fundingAmount : issue?.amount || 0)}{" "}
+            <label
+              className={`caption-small text-uppercase ${
+                !isActive ? "text-primary" : "text-white-40"
+              }`}
+            >
+              ${issue?.token?.symbol || t("common:misc.token")}
+            </label>
+          </span>
+        </div>
+        {(issue?.fundingAmount > 0) && (
+          <>
+            <div className="p-0 col-md-6 mt-1">
+              <div className="bg-dark-gray w-100 issue-funding-progress">
+                <div
+                  className={`${
+                    percentage !== 100 ? "bg-primary" : "bg-success"
+                  } issue-funding-progress`}
+                  style={{ width: `${percentage}%` }}
+                />
+              </div>
+            </div>
+            <div
+              className={`issue-percentage-text caption-small py-0 pe-0 ps-1 pb-1 col-md-2 ${
+                percentage !== 100 ? "text-white" : "text-success"
+              }`}
+            >
+              {percentage.toFixed(0)}%
+            </div>
+          </>
+        )}
       </div>
     );
   }
 
-  function renderReviews() {
+  function renderAvatar() {
+    if (issue?.creatorGithub)
```

Comment:
We can't have TWO profile pictures. We either show the one from github or we show the Identicon